css/styles.css

```diff
@@ -56,18 +57,32 @@ header h1 {
 }
 
 a {
-  color: #e81c4f;
-  border: 1px solid #e81c4f;
-  font-size: 0.6em;
-  letter-spacing: 0.2em;
-  padding: 1em 2em;
+  color: var(--red);
   text-transform: uppercase;
   text-decoration: none;
+}
+
+nav a {
+  border: 1px solid var(--red);
+  font-size: 0.6em;
+  letter-spacing: var(--h-kerning);
+  padding: 1em 2em;
   transition: none 200ms ease-out;
   transition-property: color, background;
 }
 
-a:hover {
-  background: #e81c4f;
+nav a:hover, nav a:focus, .post a:focus {
+  background: var(--red);
   color: white;
 }
+
+.post {
+  letter-spacing: var(--h-kerning);
+  border: 1px solid var(--red);
+  padding: 1em 1em;
```

Comment:
These are actually all changes I wanted to make too

Reply:
Ack